src/tools/analyzer.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+permalink: /tools/analyzer
+title: "Static Analyzer"
+description: "The tool that analyzes and validates your Dart code."
+---
+
+The static analyzer evaluates your Dart code,
+checking for errors and warnings that are specified in the
+[Dart Language Specification](https://www.dartlang.org/docs/spec/).
+
+Many tools in the Dart ecosystem use the
+[analyzer package](https://pub.dartlang.org/packages/analyzer) to
+perform static analysis:
+
+* Some IDEs (such as [WebStorm](https://webdev.dartlang.org/tools/webstorm),
+  [IntelliJ](https://www.dartlang.org/tools/jetbrains-plugin),
+  [Atom](https://atom.io/packages/dartlang))
+* [Dart Dev Compiler (DDC)](https://github.com/dart-lang/sdk/tree/master/pkg/dev_compiler)&mdash;Experimental development tool and transpiler for web apps
```

Comment:
I think `:` would be better than `—` here.
Also, make sure the capitalization is consistent.